site/en/blog/devtools-tips-container-queries/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: >
+  DevTools Tips: Inspect CSS container queries
+description: >
+  Inspect and debug CSS container queries with DevTools.
+layout: 'layouts/blog-post.njk'
+date: 2022-06-30
+authors:
+  - sofiayem
+hero: ''
+alt: >
+  DevTools Tips hero logo
+tags:
+  - css
+  - devtools
+  - devtools-tips
+---
+
+Container queries are the new CSS feature that web developers are excited about. It may trigger a tectonic shift in the design approach of website layouts, from device-based to device-agnostic.
```

Comment:
```suggestion
[CSS container queries](https://web.dev/new-responsive/#responsive-to-the-container) are the new CSS feature that web developers are excited about. It may trigger a tectonic shift in the design approach of website layouts, from page-based responsive design to container-based [responsive design](https://web.dev/new-responsive/#responsive-to-the-container).
```